core/types/celo_transaction.go

```diff
@@ -36,6 +34,6 @@ func CompareWithRates(a, b *Transaction, goldBaseFee *big.Int, ratesAndFees *exc
 	// Compare tips if effective tips and fee caps are equal
 	tipCapA := a.inner.gasTipCap()
 	tipCapB := b.inner.gasTipCap()
-	result, _ := exchange.CompareValue(rates, tipCapA, a.inner.feeCurrency(), tipCapB, b.inner.feeCurrency())
-	return result
+	c2, _ := exchange.CompareValue(rates, tipCapA, a.inner.feeCurrency(), tipCapB, b.inner.feeCurrency())
```

Comment:
Why not just assign `c, _ = exchange.CompareValue`, why a new variable?

Reply:
sure, I guess personal preference to, while possible, maintain one name per value, but in this case  it's ok. Changed it.